helix-term/src/ui/menu.rs

```diff
@@ -18,6 +18,9 @@ pub trait Item: Sync + Send + 'static {
     /// Additional editor state that is used for label calculation.
     type Data: Sync + Send + 'static;
 
+    /// Second ordering rule on equals items score
+    type TieBreaker;
+
```

Comment:
I think you should be able to add a default value here so we don't have to define it in every single implementation.